lib/api/objectDelete.js

```diff
@@ -176,9 +180,11 @@ function objectDelete(authInfo, request, log, cb) {
                             deleteInfo.removeDeleteMarker = true;
                         }
                         return services.deleteObject(bucketName, objectMD,
-                            objectKey, delOptions, log, (err, delResult) =>
-                            next(err, bucketMD, objectMD, delResult,
-                                deleteInfo));
+                            objectKey, delOptions, log, isExpiration ?
+                                's3:LifecycleExpiration:DeleteMarkerCreated' :
+                                's3:ObjectRemoved:DeleteMarkerCreated',
```

Comment:
Thiss happens when we are _removing_ a delete marker : should this not be a `s3:ObjectRemoved:Delete` event?
(or should this be done a smarter way, to avoid generating 2 notifs?)

Reply:
solved in https://github.com/scality/cloudserver/pull/5135